<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="loginPage">
       <property name="styleSheet">
        <string notr="true">#loginPage {
border-image: url(:/image/91275839_p0.jpeg);
}</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>283</y>
          <width>776</width>
          <height>37</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="loginButton">
           <property name="minimumSize">
            <size>
             <width>260</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid rgb(24, 103, 155);
border-radius:5px;
background-color: rgb(124, 203,255);
color:white;</string>
           </property>
           <property name="text">
            <string>Submit</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>150</y>
          <width>776</width>
          <height>123</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QFrame" name="formFrame">
           <property name="styleSheet">
            <string notr="true">#formFrame {
border: 1px solid rgb(24,103,155);
border-radius: 5px;
background-color: white;
}
QLabel{
color:black;
}
</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <property name="horizontalSpacing">
             <number>-1</number>
            </property>
            <property name="verticalSpacing">
             <number>-1</number>
            </property>
            <property name="leftMargin">
             <number>25</number>
            </property>
            <property name="topMargin">
             <number>25</number>
            </property>
            <property name="rightMargin">
             <number>25</number>
            </property>
            <property name="bottomMargin">
             <number>25</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Username: </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="userInput">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Password: </string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="passwordInput">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>25</height>
               </size>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="dashboardPage">
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QFrame" name="verticalFrame_3">
            <layout class="QVBoxLayout" name="verticalLayout_20"/>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QFrame" name="verticalFrame">
            <layout class="QVBoxLayout" name="verticalLayout_22"/>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QFrame" name="chart1">
            <layout class="QVBoxLayout" name="verticalLayout_13"/>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QFrame" name="chart3">
            <layout class="QVBoxLayout" name="verticalLayout_21"/>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QFrame" name="verticalFrame_2">
            <layout class="QVBoxLayout" name="verticalLayout_23"/>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QFrame" name="chart2">
            <layout class="QVBoxLayout" name="verticalLayout_19"/>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
